### PR TITLE
Add missing translations

### PR DESCRIPTION
**Pull request by tonipel**
## Description

Add missing translations

## Context

[PV-905](https://helsinkisolutionoffice.atlassian.net/browse/PV-905)

## How Has This Been Tested?

Tested manually.

## Screenshots

[PV-905]: https://helsinkisolutionoffice.atlassian.net/browse/PV-905?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/i18n/sv.json [hunk 1]
```diff
@@ -126,6 +126,7 @@
   "common.permit.Permit.discount": "Du har rätt till rabatt på gatuparkering när du betalar med mobilappar.",
   "common.permit.Permit.editVehicle": "Redigera fordonsinformation",
   "common.permit.Permit.updateCard": "Redigera dina betalkortsuppgifter",
+  "common.permit.Permit.extendPermit": "Extend parking permit",
```
Comment (mhieta) [inline src/i18n/sv.json:129]: @tonipel Maybe put swedish text here, like: `Förläng parkeringstillstånd`

**Review by mhieta — CHANGES_REQUESTED** [review]
Please update swedish translation.

File: src/i18n/sv.json (hunk 1 above)
Comment (tonipel) [reply src/i18n/sv.json]: Fixed.

**Review by mhieta — APPROVED** [review]
LGTM!

**Pull request merged**